packages/components-native/src/ErrorMessageWrapper/ErrorMessageWrapper.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ReactNode, useEffect, useRef } from "react";
+import { View, ViewStyle } from "react-native";
+import { v4 } from "react-native-uuid";
+import { useErrorMessageContext } from "./context";
+import { styles } from "./ErrorMessageWrapper.style";
+import { Icon } from "../Icon";
+import { Text } from "../Text";
+
+type WrapForTypes = "card" | "default";
+
+interface ErrorMessageWrapperProps {
+  /**
+   * The message that shows up below the children
+   */
+  readonly message?: string;
+
+  /**
+   * Changes how it gets laid out on the UI
+   */
+  readonly wrapFor?: WrapForTypes;
+
+  readonly children: ReactNode;
+}
+
+const wrapForStyle: Record<WrapForTypes, ViewStyle | undefined> = {
+  card: styles.wrapForCard,
+  default: undefined,
+};
+
+/**
+ * Adds an error message below the children but ensure the message gets read
+ * out first.
+ *
+ * This component is internal to Atlantis and shouldn't be used outside of it.
```

Comment:
This comment... scares me haha. Do we know the context behind it?

Reply:
Probably the same thing as `Typography`. It is an Atlantis component, but should be only be used internally. Even though I noticed that there's a component in jobber-mobile using it 🤔

Reply:
Right - in `PaymentMethodCreateOverlay` right?

There is [this comment](https://github.com/GetJobber/jobber-mobile/pull/4456#discussion_r935667449) from @darryltec  original PR - any updated thoughts on that @rodrigoeidelvein @darryltec ?

Reply:
I mean, the people can use it if they understand what it's doing